tests/orchestrator/test_cli_runtime.py

```
@@ -0,0 +1,258 @@
+    current_gitlab_token.set(None)
+    current_gitlab_host.set("gitlab.com")
+    current_anthropic_auth_token.set(None)
+    current_anthropic_base_url.set(None)
+
+    try:
+        dm._create_container("test-chat", "test-container")
+    except Exception:
+        # Some downstream code may still fail without real Docker — that is
+        # fine. The extra_env was already constructed and captured before
+        # any failure path that requires real network/skill state.
+        pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)